tools/cloud-build/daily-tests/integration-group-1.yaml

```diff
@@ -16,13 +16,11 @@
 # Current parallel execution tree, built to minimize total execution time and faster tests first.
 # ├── build_ghpc
 # └── fetch_builder
-#    ├── hpc-high-io (group 1)
+#    └── hpc-high-io-v4
+#        └── hpc-high-io-v5
+#           └── batch-mpi
```

Comment:
nit: batch-mpi is no longer here, remove line 21?

Reply:
Done